.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Don't Open An Issue On This Repo
+about: Don't Open An Issue On This Repo
+title: '[BUG] <title>'
+labels: Bug, Needs Triage
+assignees: ''
+
+---
+
+<!--
+Note: Please Don't Open An Issue On This Repo. If you are experiencing an issue with the SDK, open an issue with the language specific AWS SDK repo that you are using.
+For example, if you are using the GO V2 SDK, open an issue on [that repo.](https://github.com/aws/aws-sdk-go-v2/issues)
```

Comment:
Besides GO, would it be helpful to list out the links of all SDK repos we support below?